src/hexer/nifcgen.nim

```diff
@@ -1078,7 +1085,12 @@ proc traverseStmt(e: var EContext; c: var Cursor; mode = TraverseAll) =
       inc c
       e.loop c:
         traverseExpr e, c
-    of EmitS, AsgnS, RetS:
+    of EmitS:
+      e.dest.add c
+      inc c
+      e.loop c:
+        traverseExpr e, c, {efKeepStringLit}
```

Comment:
If you special case `c.kind == StringLit` here in this loop you can avoid the efKeepStringLit flag.

Reply:
okay